store/src/commonMain/kotlin/org/mobilenativefoundation/store/store5/Fallback.kt

```diff
@@ -0,0 +1,18 @@
+package org.mobilenativefoundation.store.store5
+
+/**
+ * Represents a fallback data source.
+ */
+interface Fallback<Key : Any, out Output : Any> {
+    val name: String
+    suspend fun get(key: Key): FallbackResponse<Output>
+}
+
+sealed class FallbackResponse<out Output : Any> {
```

Comment:
is error possible?

Reply:
Removed! This interface and class should have been removed in the refactor away from Superstore and Warehouse